async function main() {}
console.log('test35e3ttaataaaaacaaaaaaaaaaaaaaaa2');
main()
  .then(() => console.log('done'))
  .catch((e) => console.error(e));
